docs/detections/add-exceptions.asciidoc

```diff
@@ -13,6 +13,8 @@ You can add exceptions to a rule from the rule details page, the Alerts table, t
 * To ensure an exception is successfully applied, ensure that the fields you've defined for its query are correctly and consistently mapped in their respective indices. Refer to {ecs-ref}[ECS] to learn more about supported mappings.
 
 * Be careful when adding exceptions to <<create-eql-rule,event correlation>> rules. Exceptions are evaluated against every event in the sequence, and when the exception matches _all_ event(s) in the sequence, alerts _are not_ generated. If the exception only matches _some_ of the events in the sequence, alerts _are_ generated.
+
+* Please note that rule exception definitions are **case-sensitive**. You need to configure the arguments exactly as they appear in the log lines as otherwise the rule exception won't match.
```

Comment:
I need to follow up here - there are`.caseless` fields that would not abide by this statement so need to double check that functionality.

Reply:
Sorry this fell through. Confirmed some things about .caseless. EQL functions as case insensitive, but our other rules, that use KQL, function as case sensitive. `.caseless` was a way for us to provide users a way of having field values evaluated as case insensitive. So the statement above is true, but we may want to note that in the event you do not want a field evaluated as case sensitive, some ECS fields do have a `.caseless` version available.

Reply:
Thanks for following up on this, @yctercero! I'll modify the note so that it includes this information.

Reply:
Added in [fa98d08](https://github.com/elastic/security-docs/pull/3860/commits/fa98d08f0c20d1f68a02c26880a388a911c3acfe)

Reply:
Looks great! Thanks for the update.